vars/util.groovy

```diff
@@ -194,4 +194,18 @@ GIT INFORMATION REPORT
     } else {
         println '[WARNING] The variable GIT_INFORMATION_REPORT does not exist'
     }
+}
+
+def getNextSnapshot(String version) {
+    String [] versionSplit = version.split("\\.")
+    
+    if(versionSplit.length == 3) {
+        int majorVersion = Integer.parseInt(versionSplit[0])
+        int minorVersion = Integer.parseInt(versionSplit[1])
+        int microVersion = Integer.parseInt(versionSplit[2])
+
+        return "${majorVersion}.${minorVersion}.${microVersoin+1}-SNAPSHOT"
```

Comment:
```suggestion
        return "${majorVersion}.${minorVersion}.${microVersion+1}-SNAPSHOT"
```

Reply:
Thanks for pointing out this :)

Reply:
No, but you can basically just copy and paste the one I mentioned [in this comment](https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/77#issuecomment-656504922).

Reply:
Okay thanks I did it